runtime/compiler/optimizer/J9RecognizedCallTransformer.cpp

```diff
@@ -149,18 +149,17 @@ void J9::RecognizedCallTransformer::process_java_lang_StringCoding_encodeASCII(T
    */
 
    TR::SymbolReference *methodSymRef = comp()->getSymRefTab()->findOrCreateEncodeASCIISymbol();
-   TR::Node *encodeASCIINode = TR::Node::createWithSymRef(TR::acall, 6, methodSymRef);
+   TR::Node *encodeASCIINode = TR::Node::createWithSymRef(TR::acall, 3, methodSymRef);
 
    TR::Node *newInputNode = TR::Node::create(sourceArrayNode, comp()->target().is64Bit() ? TR::aladd : TR::aiadd, 2);
-   TR::Node *newInputChild2Node = TR::Node::create(sourceArrayNode, comp()->target().is64Bit() ? TR::lconst : TR::iconst, 0, (int32_t)TR::Compiler->om.contiguousArrayHeaderSizeInBytes());
+   TR::Node *arrayHeaderSizeNode = TR::Node::create(sourceArrayNode, comp()->target().is64Bit() ? TR::lconst : TR::iconst, 0, (int32_t)TR::Compiler->om.contiguousArrayHeaderSizeInBytes());
```

Comment:
Should you just use the `lconst` or `iconst` routines on the Node rather than potentially pass in a 32-bit value when you create an `lconst` node instance ?

Reply:
Switched to using `lconst` and `iconst` routines here: 0e36ab6f7961b3c5a372d4e1b269f35c29515f17